### DISCUSSION TITLE
Use it with your own photos

### DISCUSSION THREAD

**Discussion by leoavila1619** [lllyasviel/Fooocus#2703] [Q&A] [answered]
Hello everyone can I use this program to edit my own photos? Can you help what is the way to do it please. I appreciate it

**Comment by ILikeToasters** [comment, marked as answer]
Not traditional editing but you can use real photos with inpainting for example. If you wanted to remove or generate something in the photo that is possible. You can also use them for image prompts. This might give you an idea: https://youtu.be/rMQQx6LLGSo
